plasmapy/physics/transport.py

```diff
@@ -1003,7 +1004,8 @@ def mobility(T,
 
     References
     ----------
-    .. [1] https://en.wikipedia.org/wiki/Electrical_mobility#Mobility_in_gas_phase
+    .. [1] https://en.wikipedia.org/wiki/
+           Electrical_mobility#Mobility_in_gas_phase
```

Comment:
Same thing may apply here. You should be able to check this in the output of CircleCI's html job.